# Do not install sphynx, as documantation is not made (not even the source bundled)
# October 11th, 2017 by B. Hajgato (Free University Brussel - VUB) 
--- dammit-0.3.2/setup.py.orig	2016-11-23 05:49:45.000000000 +0100
+++ dammit-0.3.2/setup.py	2017-10-11 22:28:09.104110519 +0200
@@ -37,8 +37,6 @@
                                 'pandas>=0.18.1',
                                 'khmer>=2.0',
                                 'doit==0.29.0',
-                                'Sphinx>1.3.1',
-                                'sphinx-rtd-theme>=0.1.9',
                                 'nose==1.3.4',
                                 'nose-capturestderr==1.0',
                                 'ficus>=0.1',
--- dammit-0.3.2/dammit.egg-info/requires.txt.orig	2016-11-23 05:51:53.000000000 +0100
+++ dammit-0.3.2/dammit.egg-info/requires.txt	2017-10-11 22:26:59.426080680 +0200
@@ -2,8 +2,6 @@
 pandas>=0.18.1
 khmer>=2.0
 doit==0.29.0
-Sphinx>1.3.1
-sphinx-rtd-theme>=0.1.9
 nose==1.3.4
 nose-capturestderr==1.0
 ficus>=0.1
